.golangci.yml

```diff
@@ -1,62 +1,63 @@
-# Please refer to the official golangci-lint config documentation for more details:
-# https://golangci-lint.run/usage/configuration/
-# https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml
+version: "2"
 run:
-  timeout: 10m
   tests: true
-  # default is true. Enables skipping of directories:
-  #   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
-  exclude-dirs-use-default: true
-
 linters:
-  disable-all: true
+  default: none
   enable:
-    - goimports
-    - gosimple
+    - bidichk
+    - copyloopvar
+    - durationcheck
+    - gosec
     - govet
     - ineffassign
     - misspell
+    - revive
+    - staticcheck
     - unconvert
-    - typecheck
     - unused
-    - staticcheck
-    - bidichk
-    - durationcheck
-    - copyloopvar
     - whitespace
-    - gosec
-    - revive
-
-    # - structcheck # lots of false positives
-    # - errcheck #lot of false positives
-    # - contextcheck
-    # - errchkjson # lots of false positives
-    # - errorlint # this check crashes
-    # - exhaustive # silly check
-    # - makezero # false positives
-    # - nilerr # several intentional
-
-linters-settings:
-  gofmt:
-    simplify: true
-  gosec:
-    excludes:
-      - G115
-      - G406 # ignore ripe160 deprecation
-      - G507 # ignore ripe160 deprecation
-  revive:
+  settings:
+    gosec:
+      excludes:
+        - G115
+        - G406
+        - G507
+    revive:
+      rules:
+        - name: var-naming
+          arguments:
+            - []
+            - []
+            - - upperCaseConst: true
+          severity: warning
+          disabled: false
+          exclude:
+            - ""
+  exclusions:
```

Comment:
Looks like [exclusions](https://golangci-lint.run/usage/configuration/#linters-configuration) should be a top level config in v2.